module/actor.js

```diff
@@ -112,6 +112,7 @@ class DCCActor extends Actor {
       capLevel: false,
       maxLevel: 0,
       rollAttackBonus: false,
+      rollABWithAttack: false,
```

Comment:
Bit of a nit pick, but the variable name uses the abbreviation "AB" whereas the line before spells it out. AB isn't a super common abbreviation, so consider renaming the variable `rollAttackWithAttackBonus`?